### ISSUE TITLE
Bower is deprecated

### ISSUE THREAD

**Issue by jancborchardt** [nextcloud/mail#384]
On `make optimize-js` I recently got this message:
```
npm WARN deprecated bower@1.8.0:
..psst! While Bower is maintained, we recommend Yarn and Webpack for *new* front-end projects!
Yarn's advantage is security and reliability, and Webpack's is support for both CommonJS and AMD projects.
Currently there's no migration path but we hope you'll help us figure out one.
```

Well who would have thought. ;) We have to choose a new cool tool. cc @nextcloud/mail

**Comment by jancborchardt** [comment]
Also cc @georgehrke @tcitworld @raghunayyar of Calendar and @irgendwie @Henni @skjnldsv of Contacts where Bower is also used. How do you deal with the deprecation there?

**Comment by georgehrke** [comment]
> Also cc @georgehrke @tcitworld @raghunayyar of Calendar and @irgendwie @Henni @skjnldsv of Contacts where Bower is also used. How do you deal with the deprecation there?

Not at all at the moment. AFAIK it will still be supported for a while.

We will switch to vue/react/or whatever we decide for in [1] and also migrate to webpack and npm then.


[1] https://help.nextcloud.com/t/give-vue-2-x-a-chance/9956/12

**Comment by ChristophWurst** [comment]
Fixed by #468
